database/schema/ve_db_postgresql.sql

```diff
@@ -579,6 +579,36 @@ CREATE INDEX ON deleted_systems(when_deleted);
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON deleted_systems TO ve_db_user_listener;
 
+
+-- repo
+CREATE TABLE IF NOT EXISTS repo (
+  id SERIAL,
+  name TEXT NOT NULL UNIQUE, CHECK (NOT empty(name)),
+  PRIMARY KEY (id)
+) TABLESPACE pg_default;
+
+CREATE INDEX ON repo(name);
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON repo TO ve_db_user_listener;
+
+
+-- system_repo
+CREATE TABLE IF NOT EXISTS system_repo (
+  inventory_id TEXT NOT NULL, CHECK (NOT empty(inventory_id)),
+  repo_id INT NOT NULL,
+  UNIQUE (inventory_id, repo_id),
+  CONSTRAINT inventory_id
+    FOREIGN KEY (inventory_id)
+    REFERENCES system_platform (inventory_id),
+  CONSTRAINT repo_id
+    FOREIGN KEY (repo_id)
+    REFERENCES repo (id)
+) TABLESPACE pg_default;
+
+CREATE INDEX ON system_repo(inventory_id);
```

Comment:
I'd also add index on `repo_id` column to quickly list systems subscribed to single repo. This will be useful in next step.